### PR TITLE
readme has been updated

### PR DESCRIPTION
**Pull request by sf-jeff-lim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,143 @@
...
-## Installation
+<br />
+
+## Introduction
+
+**Sendbird Desk** enables strong customer engagement through live, in-app support. The Desk SDK lets you easily initialize, configure, and build customer support-related functionality into your Android applications.
+
+### How it works
+
+Sendbird Desk is a plugin of the [Sendbird Chat Platform](https://sendbird.com/docs/chat) for managing tickets, and thus Desk events are handled by event handlers through the [Chat SDK](https://github.com/sendbird/sendbird-ios-framework).
```
Comment (hoon-sung) [inline README.md]: Chat SDK links to iOS

File: README.md [hunk 2]
```diff
@@ -1,67 +1,143 @@
...
 
-And then add the following lines to your app-level `build.gradle` file.
-```gradle
+#### - App Level
+
+```bash
+// Add the following dependency to the build.gradle file at the app level.
 dependencies {
     implementation 'com.sendbird.sdk:sendbird-android-sdk:3.0.141'
     implementation 'com.sendbird.sdk:sendbird-desk-android-sdk:1.0.8'
```
Comment (hoon-sung) [inline README.md]: The latest versions for Messaging SDK and Desk SDK are `3.0.151`, `1.0.9` respectively.

File: README.md (hunk 1 above)
Comment (sf-jeff-lim) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (sf-jeff-lim) [reply README.md]: Done!

File: README.md [hunk 3]
```diff
@@ -73,314 +149,510 @@ public class MyApplication extends Application {
...
+|State|Description|
+|---|---|
+|WAITING|Set when an agent sends a customer feedback request message.|
+|CONFIRMED|Set when a customer sends a response.|
 
+When a customer replies to the message, their score and comment for the ticket are sent to the Desk server by calling the `Ticket.submitFeedback()` method. Then, the state of the confirmation request message is changed to `CONFIRMED`.
 
 ```java
-ticket.setTicketPriority(priority, new Ticket.SetTicketPriorityHandler() {
+Ticket.submitFeedback(USER_MESSAGE, SCORE, new Ticket.SubmitFeedbackHandler() {
```
Comment (hoon-sung) [inline README.md]: The code snippet here should be 
```
ticket.submitFeedback(USER_MESSAGE, SCORE, COMMENT, net Ticket.SubmitFeedbackHandler() {
    @Override
    public void onResult(Ticket ticket, SendBirdException e) {
        if (e != null) {    // Error.
            return;
        }
        ...   
    }
});
```

File: README.md [hunk 4]
```diff
@@ -73,314 +149,510 @@ public class MyApplication extends Application {
...
 
-Ticket information could be kept in `Priority`. 
-`setTicketPriority()` in `Ticket` lets the SDK set the `Priority` of the current ticket. 
-If you didn't set the Priority of ticket the default `Priority` has been set to `Priority.MEDIUM`
+|State|Description|
+|---|---|
+|WAITING|Set when an agent sends a customer feedback request message.|
+|CONFIRMED|Set when a customer sends a response.|
 
+When a customer replies to the message, their score and comment for the ticket are sent to the Desk server by calling the `Ticket.submitFeedback()` method. Then, the state of the confirmation request message is changed to `CONFIRMED`.
```
Comment (hoon-sung) [inline README.md]: `Ticket.submitFeedback()` -> `ticket.submitFeedback()` (lowercased t as `submitFeedback()` is not a static method)

**Review by hoon-sung — COMMENTED** [review]
Previous comments were all fixed, but I've found a mistake in the code snippet (I've missed it on previous review)

File: README.md (hunk 3 above)
Comment (sf-jeff-lim) [reply README.md]: Done!

File: README.md (hunk 4 above)
Comment (sf-jeff-lim) [reply README.md]: Done!

**Review by hoon-sung — APPROVED** [review]
Sorry I've missed

**Pull request merged**